### ISSUE TITLE
LOG_LIMIT in fuzz_opt.py

### ISSUE THREAD

**Issue by tlively** [WebAssembly/binaryen#2231]
What benefit does LOG_LIMIT have? When I'm trying to use the fuzzer I have to set it to some effectively infinite value if I want to be able to reproduce the errors the fuzzer finds. I propose we get rid of it.

**Comment by kripken** [comment]
Fair enough. Sometimes those lines are annoyingly long when just trying to read the non-error output. But yeah, sounds good to remove it.

**Comment by tlively** [comment]
Looks like LOG_LIMIT no longer appears in the code base.
